docs/installation/setup-ide.md

```diff
@@ -70,15 +70,26 @@ The following Visual Studio configuration shows a typical configuration.
 {#setup-rider}
 ## Setup Rider
 
-% TODO
-```{admonition} Rider plugin is not available yet
-:class: error
-
-The Reqnroll Rider plugin has not yet been ported and released. Please, come back later or help contribute to it in our [open-source GitHub project](https://github.com/reqnroll/Reqnroll.Rider).
+In order to use Reqnroll with Rider, you need to install the [Reqnroll for Rider](https://plugins.jetbrains.com/plugin/24012-reqnroll-for-rider) extension.
 
-We recommend using the Visual Studio 2022 or the Visual Studio Code integration until the plugin will be ready. The SpecFlow Rider plugin can also be used with limited capabilities.
+```{warning}
+The *Reqnroll with Rider* extension cannot work together with the *SpecFlow for Rider* extension, as they both process feature files. As the Reqnroll extension also supports SpecFlow projects, you can remove the SpecFlow extension if you install the Reqnroll extension. Alternatively, you can disable the SpecFlow extension for the time you work with Reqnroll. 
 ```
 
+1. Open Rider and verify that you are using a compatible version. The following versions are verified to work with Reqnroll:
+    - `2024.1.4`
+    - `2024.2 EAP9 build 242.20224.162`
+2. Top right of Rider click the gear icon and press plugins.
+3. Click `Marketplace`. 
+3. Enter `Reqnroll` in the search box and install.
+4. Enter `Gherkin` in the search box and install.
+5. Open csproj and verify your project contains 
```

Comment:
It would be a good thing to list this as a workaround for a known problem https://github.com/reqnroll/Reqnroll.Rider/issues/1

Reply:
![image](https://github.com/user-attachments/assets/0111c193-f012-4994-b884-cd9cef03cb87)
added link to issue ticket